gym/utils/env_checker.py

```diff
@@ -165,47 +196,49 @@ def check_reset_options(env: gym.Env):
                 f"The error was: {e}"
             )
     else:
-        raise error.Error(
+        raise gym.error.Error(
             "The `reset` method does not provide the `options` keyword argument"
         )
 
 
-def check_render(env: gym.Env, warn: bool = True):
-    """Check the declared render modes/fps of the environment.
-
-    Args:
-        env: The environment to check
-        warn: Whether to output additional warnings
-    """
-    render_modes = env.metadata.get("render_modes")
-    if render_modes is None:
-        if warn:
+def check_space_limit(space, space_type: str):
```

Comment:
Because this error will be raised for all mujoco environments (as they obs and action spaces of `np.inf`) therefore, I wished this warning to only occur in `env_checker` and not in the passive version. However Im happy to move it back to the passive version.